nat-lab/pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "nat-lab"
 
 [tool.black]
 preview = true
-exclude="tests/protobuf"
+exclude = "tests/protobuf,tests/uniffi"
```

Comment:
Will this not make all the python tools skip `libtelio_remote.py` and `libtelio_proxy.py`?

Reply:
you're right, will change so it would only skip generated part

Reply:
fixed